spring-test/src/test/java/org/springframework/test/context/support/BootstrapTestUtilsMergedConfigTests.java

```diff
@@ -271,10 +270,7 @@ public void buildMergedConfigForContextHierarchy() {
 
 		MergedContextConfiguration parent = mergedConfig.getParent();
 		assertThat(parent).as("parent config").isNotNull();
-		// The following does not work -- at least not in Eclipse.
```

Comment:
I'm not sure why it didn't work, but considering that Spring Boot doesn't use the `AssertionsForClassTypes` at all, I expected that it's okay now.

Reply:
@sbrannen would you mind taking a look at this in Eclipse?

Reply:
Surprise! The Eclipse compiler no longer has a issue with that. 👍